2022q3/cloud-init.adoc

```diff
@@ -0,0 +1,22 @@
+=== FreeBSD as a Tier I cloud-init Platform
+
+Links: +
+link:https://https://cloud-init.io/[cloud-init Website] URL: link:https://cloud-init.io/[https://cloud-init.io/] +
+link:https://cloudinit.readthedocs.io/en/latest/[cloud-init Documentation] URL: link:https://cloudinit.readthedocs.io/en/latest/[https://cloudinit.readthedocs.io/en/latest/] +
+link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[cloud-init ongoing refactorization] URL: link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst[link:https://github.com/canonical/cloud-init/blob/main/WIP-ONGOING-REFACTORIZATION.rst] +
+
+Contact: Mina Galić <me+FreeBSD@igalic.co>
+
+cloud-init is the standard way of provisioning servers in the cloud.
+Unfortunately, cloud-init’s support for operating systems other than Linux is rather poor and the lack of cloud-init support on FreeBSD is a hindrance to cloud providers who want to offer FreeBSD as a Tier I platform.
+To remedy this, this project aims to bring FreeBSD cloud-init support on par with Linux support. Though the plan is to lift support across all BSDs.
```

Comment:
s/support. Though/support, although/ if you mean you're focusing on FreeBSD first but making sure other BSDs also benefit from your work, even if less or indirectly.
s/Though/However/ if you're intend to enhance all BSDs more or less equally and are just mentioning FreeBSD first because it's what this status report covers.
If you don't mean either, please explain.

Reply:
I don't think i can do all BSDs justice, but they will all definitely benefit from my work

Reply:
*nod* first variant then, I think.